src/program.ts

```diff
@@ -2996,8 +2998,8 @@ export class File extends Element {
 
   /** Creates an imported namespace from this file. */
   asAliasNamespace(
-    name: string, 
-    parent: Element, 
+    name: string,
+    parent: Element,
```

Comment:
These had extra spaces 🤔  is that intentional?

Reply:
Not intentional, would guess this somehow slipped through.